### PR TITLE
Show how to import the flutter_markdown library

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
cc @Hixie

**Comment by Hixie** [comment]
LGTM for now but cc @abarth who I believe has plans to move this elsewhere.

File: packages/flutter_markdown/lib/flutter_markdown.dart [hunk 1]
```diff
@@ -3,6 +3,8 @@
 // found in the LICENSE file.
 
 /// A library to render markdown formatted text.
+///
+/// To use, import `package:flutter/flutter_markdown.dart`.
```
Comment (abarth) [inline packages/flutter_markdown/lib/flutter_markdown.dart]: This is incorrect.  The correct import is:

```package:flutter_markdown/flutter_markdown.dart```

**Comment by abarth** [comment]
Also, please don't push to branches in the main repository.  Instead, push to a branch in your fork and send the pull request from there.  We use that process so that folks with edit rights to the repo use the same process as folks who do not yet have edit rights.

**Comment by sethladd** [comment]
Ah, good call, thanks. I'll close this and do it from a fork.

**Pull request closed without merging**